compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -456,13 +456,9 @@ extern "C" LLVMTargetMachineRef LLVMRustCreateTargetMachine(
   Options.UseInitArray = UseInitArray;
 
 #if LLVM_VERSION_LT(17, 0)
-  if (ForceEmulatedTls) {
-    Options.ExplicitEmulatedTLS = true;
-    Options.EmulatedTLS = true;
-  }
-#else
-  Options.EmulatedTLS = ForceEmulatedTls || Trip.hasDefaultEmulatedTLS();
+  Options.ExplicitEmulatedTLS = UseEmulatedTls;
```

Comment:
```suggestion
  Options.ExplicitEmulatedTLS = true;
```
This option is a bit confusing, essentially it means "override the target default".